app/root.css

```diff
@@ -43,7 +43,7 @@
   --spacing-104: clamp(56px, 10.6vw, 104px);
   --spacing-128: clamp(56px, 13.8vw, 128px);
   --spacing-192: clamp(80px, 17vw, 192px);
-  --spacing-288: clamp(128px, 25.6vw, 288px);
+  --spacing-288: clamp(128px, 20vw, 288px);
```

Comment:
I'm still uncertain about making the spacing variables vw. I use a system based on 8 to make it clean. But let's try it and see.

I definitely think we should make the columns vw with clamp, which I believe is the case now!